types/dialogify-browser/Dialogify.d.ts

```diff
@@ -0,0 +1,238 @@
+/// <reference types="jquery" />
```

Comment:
please rename file using 'git mv' , e.g. `git mv ./Dialogify.d.ts ./dialogify.d.ts`.
File in the module is all lower case

Reply:
Fixed!